corehq/apps/hqadmin/management/commands/record_deploy_success.py

```diff
@@ -85,6 +88,10 @@ def handle(self, *args, **options):
 
         if options['mail_admins']:
             message_body = get_deploy_email_message_body(
-                environment=options['environment'], user=options['user'],
-                compare_url=compare_url)
+                user=options['user'],
+                compare_url=compare_url,
+                current_deploy=options.get('current_deploy'),
+                previous_deploy=options.get('current_deploy'),
```

Comment:
yeah, gonna close out this PR though and go with this one instead: https://github.com/dimagi/commcare-hq/pull/11389